shared/chat/conversation/list.desktop.js

```diff
@@ -76,8 +76,10 @@ class ConversationList extends Component<void, Props, State> {
   componentWillUpdate (nextProps: Props, nextState: State) {
     // If a message has moved from pending to sent, tell the List to discard
     // heights for it (which will re-render it and everything after it)
+    // TODO this doesn't work for things that take a bit to load (imgs)
```

Comment:
Unrelated to this PR but maybe all our images should have width, height set even before being downloaded?
It's a huge bummer when images load and everything on a web pages gets shifted.